omnigibson/sensors/vision_sensor.py

```diff
@@ -433,47 +532,30 @@ def _obs_space_mapping(self):
-            gym.spaces.Box(low=-np.inf, high=np.inf, shape=(), dtype=np.int32),  # uniqueId
-            gym.spaces.Text(min_length=1, max_length=50, charset=VALID_OMNI_CHARS),  # name
-            gym.spaces.Text(min_length=1, max_length=50, charset=VALID_OMNI_CHARS),  # semanticLabel
-            gym.spaces.Text(min_length=0, max_length=50, charset=VALID_OMNI_CHARS),  # metadata
-            gym.spaces.Sequence(space=gym.spaces.Box(low=0, high=MAX_INSTANCE_COUNT, shape=(), dtype=np.uint)), # instanceIds
-            gym.spaces.Box(low=0, high=MAX_CLASS_COUNT, shape=(), dtype=np.uint32),  # semanticId
+            gym.spaces.Box(low=0, high=np.iinfo(np.uint32).max, shape=(), dtype=np.uint32),  # semanticId
             gym.spaces.Box(low=0, high=MAX_VIEWER_SIZE, shape=(), dtype=np.int32),  # x_min
             gym.spaces.Box(low=0, high=MAX_VIEWER_SIZE, shape=(), dtype=np.int32),  # y_min
             gym.spaces.Box(low=0, high=MAX_VIEWER_SIZE, shape=(), dtype=np.int32),  # x_max
             gym.spaces.Box(low=0, high=MAX_VIEWER_SIZE, shape=(), dtype=np.int32),  # y_max
+            gym.spaces.Box(low=-1.0, high=1.0, shape=(), dtype=np.float32),  # occlusion ratio
         )))
 
-        camera_space = gym.spaces.Dict(dict(
-            pose=gym.spaces.Box(low=-np.inf, high=np.inf, shape=(4, 4), dtype=float),
-            fov=gym.spaces.Box(low=0, high=np.inf, shape=(), dtype=float),
-            focal_length=gym.spaces.Box(low=0, high=np.inf, shape=(), dtype=float),
-            horizontal_aperture=gym.spaces.Box(low=0, high=np.inf, shape=(), dtype=float),
-            view_projection_matrix=gym.spaces.Box(low=-np.inf, high=np.inf, shape=(4, 4), dtype=float),
-            resolution=gym.spaces.Dict(dict(
-                width=gym.spaces.Box(low=1, high=MAX_VIEWER_SIZE, shape=(), dtype=np.uint),
-                height=gym.spaces.Box(low=1, high=MAX_VIEWER_SIZE, shape=(), dtype=np.uint),
-            )),
-            clipping_range=gym.spaces.Box(low=0, high=np.inf, shape=(2,), dtype=float),
-        ))
-
         obs_space_mapping = dict(
             rgb=((self.image_height, self.image_width, 4), 0, 255, np.uint8),
-            depth=((self.image_height, self.image_width), 0.0, 1.0, np.float32),
+            depth=((self.image_height, self.image_width), 0.0, np.inf, np.float32),
             depth_linear=((self.image_height, self.image_width), 0.0, np.inf, np.float32),
-            normal=((self.image_height, self.image_width, 3), -1.0, 1.0, np.float32),
-            seg_semantic=((self.image_height, self.image_width), 0, MAX_CLASS_COUNT, np.uint32),
+            normal=((self.image_height, self.image_width, 4), -1.0, 1.0, np.float32),
+            seg_semantic=((self.image_height, self.image_width), -1, MAX_CLASS_COUNT, np.int32),
             seg_instance=((self.image_height, self.image_width), 0, MAX_INSTANCE_COUNT, np.uint32),
+            seg_instance_id=((self.image_height, self.image_width), 0, MAX_INSTANCE_COUNT, np.uint32),
```

Comment:
these MAX_CLASS_COUNT and MAX_INSTANCE_COUNT might change

Reply:
Does it make sense to make both of them the max of uint32?